perses/tests/test_repex.py

```diff
@@ -188,5 +185,218 @@ def test_RESTCapableHybridTopologyFactory_repex_charge_mutation():
                        + data['ala-arg']['error'] ** 2
                        + data['lys-ala']['error'] ** 2
                        + data['ala-lys']['error'] ** 2)
+        assert DDG < 6 * dDDG, f"DDG ({DDG}) is greater than 6 * dDDG ({6 * dDDG})"
+
+
+@pytest.mark.gpu_needed
+def test_RESTCapableHybridTopologyFactory_repex_neutral_transformation():
+    """
+    Run CCC->CCCC and CCCC->CCC repex with the RESTCapableHybridTopologyFactory and make sure that the free energies are
+    equal and opposite.
+
+    """
+
+    from pkg_resources import resource_filename
+    import numpy as np
+
+    from openmm import unit
+
+    from perses.dispersed.utils import configure_platform
+    from perses.app.relative_setup import RelativeFEPSetup
+    from perses.samplers.multistate import HybridRepexSampler
+    from perses.annihilation.relative import RESTCapableHybridTopologyFactory
+
+    from openmmtools.multistate import MultiStateReporter, MultiStateSamplerAnalyzer
+    from openmmtools import cache, utils, mcmc
+    platform = configure_platform(utils.get_fastest_platform().getName())
+
+    data = {}
+    n_iterations = 3000
+    n_states = 12
+    transformations = [(0, 1), (1, 0)]  # Ligand indices in perses/data/test.smi
+
+    with enter_temp_directory() as temp_dir:
+        for ligand_A_index, ligand_B_index in transformations:
+            # Generate topology proposal and positions
+            smiles_filename = resource_filename("perses", os.path.join("data", "test.smi"))
+            fe_setup = RelativeFEPSetup(
+                ligand_input=smiles_filename,
+                old_ligand_index=ligand_A_index,
+                new_ligand_index=ligand_B_index,
+                forcefield_files=["amber14/tip3p.xml"],
+                small_molecule_forcefield="gaff-2.11",
+                phases=["solvent"],
+                solvent_padding=1.7 * unit.nanometers)
```

Comment:
Could we use a smaller padding, like 1 nm?

Reply:
With openmm > 7.7, we cannot use a smaller padding, because otherwise we see this error: https://github.com/choderalab/perses/issues/949

Reply:
This might need to be reconsidered. I think this changed along the way, since I've been using the default value of 1.1 nm for the `PointMutationExecutor` with the openmm nightly build and it is working correctly. I'll make tests to try to come up with a good value.

Reply:
@ijpulidos : Were you able to reproduce the `NonbondedForce: The cutoff distance cannot be greater than half the periodic box size` with openmm 7.7?